ui/app/styles/_variables.scss

```diff
@@ -22,6 +22,7 @@ $button-shadow: 0 3px 2px rgba(var(--shadow), 0.2);
   --border: #{dehex(color.$ui-gray-200)};
   --outline: #{dehex(color.$ui-gray-300)};
   --card: #{dehex(color.$white)};
+  --highlighted-list-item: #{dehex(color.$ui-cool-gray-100)};
```

Comment:
Shall we add a dark mode color for this too?

<img width="364" alt="CleanShot 2021-11-24 at 12 06 15@2x" src="https://user-images.githubusercontent.com/34030/143227248-f8aff7f8-592a-4d87-a3ba-87f8de21213c.png">

Reply:
yep – I was going to call this out. It should probably be a lighter gray